ui/src/app/app-routing.tsx

```diff
@@ -101,277 +102,281 @@ interface RoutingProps {
 
 export const AppRoutingComponent: React.FunctionComponent<RoutingProps> = ({onSignIn, signIn}) => {
   const {authLoaded = false} = useStore(authStore);
-  return authLoaded && <AppRouter>
-    <AppRoute
-        path='/cookie-policy'
-        component={() => <CookiePolicyPage routeData={{title: 'Cookie Policy'}}/>}
-    />
-    <AppRoute
-        path='/login'
-        component={() => <SignInPage routeData={{title: 'Sign In'}} onSignIn={onSignIn} signIn={signIn}/>}
-    />
-    <AppRoute
-        path='/session-expired'
-        component={() => <SessionExpiredPage routeData={{title: 'You have been signed out'}} signIn={signIn}/>}
-    />
-    <AppRoute
-        path='/sign-in-again'
-        component={() => <SignInAgainPage routeData={{title: 'You have been signed out'}} signIn={signIn}/>}
-    />
-    <AppRoute
-        path='/user-disabled'
-        component={() => <UserDisabledPage routeData={{title: 'Disabled'}}/>}
-    />
 
-    <ProtectedRoutes guards={[signInGuard]}>
+  return authLoaded && <React.Fragment>
+    <NotificationModal/>
```

Comment:
Please use the hide whitespace option - the changes outside of this are almost all whitespace

This modal can be moved once we have removed Angular. It chooses when to render itself based on the `notificationStore`. This allows any component to utilize this modal for error handling and confirmation